src/Collections-Strings/String.class.st

```diff
@@ -1380,6 +1380,10 @@ String >> expandMacrosWithArguments: anArray [
 { #category : 'finding/searching' }
 String >> findAnySubstring: aCollection startingAt: start [
 	"Answer the index where an element of aCollection begins. If none are found, answer size + 1. aCollection is an Array of Strings or Characters."
+	"'abcdef' findAnySubstring: 'cde' startingAt: 1 >>> 3"
```

Comment:
Please put parentheses because >>> is a binary message so it is executed prior to findAny..: